_includes/page/links.html

```diff
@@ -34,6 +35,7 @@
 {%- comment %}
     Site resources
 {% endcomment %}
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
```

Comment:
This seems like a poor choice - adding 54kB to every page load just for a single close icon